examples/shop-page/Components/ShopPage.lua

```diff
@@ -0,0 +1,74 @@
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+
+local Components = ReplicatedStorage:WaitForChild("Components")
```

Comment:
do we need to call WaitForChild here since the component lives inside Components?

Reply:
Nope, my bad!